molo/core/templatetags/core_tags.py

```diff
@@ -272,6 +298,83 @@ def load_child_articles_for_section(context, section, count=5):
     return articles
 
 
+@register.simple_tag(takes_context=True)
+def load_tags_for_homepage(
+        context, latest_articles_count=12, section_count=1, tag_count=4,
+        sec_articles_count=4):
+    def get_positional_tag_articles(request, tag, exclude_list):
+
+        try:
+            pks = []
+            for article_tag in ArticlePageTags.objects.filter(tag=tag):
+                pks.append(article_tag.page.pk)
```

Comment:
I think we can collapse this to `[article_tag.page.pk for article_tag in ArticlePageTags.objects.filter(tag=tag)]`